test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -18,6 +19,31 @@ Describe "Get-Process" -Tags "CI" {
     It "Should have not empty Name flags set for Get-Process object" -Pending:$IsMacOS {
         $ps | foreach-object { $_.Name | Should Not BeNullOrEmpty }
     }
+
+    It "Should throw an error for non existing process id." {
```

Comment:
The new tests  do not match the PR description. Please move to new PR or correct the PR header and description.